projects/plugins/crm/src/js/components/automations-admin/components/bulk-workflow-actions/index.tsx

```diff
@@ -1,26 +1,61 @@
 import { DropdownMenu, MenuItem } from '@wordpress/components';
-import { dispatch } from '@wordpress/data';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
```

Comment:
There were a few non-internationalized strings in here, so I went ahead and corrected that.